include/mbedtls/mbedtls_config.h

```diff
@@ -3765,6 +3780,20 @@
  */
 //#define MBEDTLS_PLATFORM_GMTIME_R_ALT
 
+/**
+ * Uncomment the macro to let Mbed TLS use your alternate implementation of
+ * mbedtls_platform_buf_cache_{write,read}(). This replaces the default
+ * implementation in platform_util.c.
```

Comment:
What't the default implementation in platform_util.c ? Till now, I only see function mbedtls_platform_zeroize() and function mbedtls_platform_gmtime_r() there.

Reply:
line 146 , platform_util.c in this PR